### DISCUSSION TITLE
Installation appears stuck at Finish preparation

### DISCUSSION THREAD

**Discussion by bastianhilton** [medusajs/medusa#6010] [Q&A]
I have been attempting to install Medusajs for almost an hour now, it is currently stuck on Finish preparation step. I am installing it locally following this guide https://docs.medusajs.com/create-medusa-app. Does it normally take this long or is there something I should be looking out for?

I'm installing this locally btw on Windows 11 and connecting it to AWS RDS Postgres. I've attempted to install it via digital ocean droplet (Linux) but because of how long it takes the terminal times out.

Any help would be greatly appreciated

**Comment by bastianhilton** [comment]
i got around the issue by cancelling the installation at this step after waiting over 10 hours. I then ran npx medusa develop to start the server and everything appeared fine. 

To be noted this is still an issue however as the installation doesnt show that it is completed.

**Comment by olivermrbl** [comment]
@shahednasser, any idea what could cause this?

**Reply by shahednasser** [reply]
It would be difficult to determine the exact reason, but it might make sense to add a `--verbose` option in similar cases so that we can see the full logs for similar scenarios.

**Reply by shahednasser** [reply]
@bastianhilton can I get you to try the following snapshot with the `--verbose` option and let us know the logs you see in the terminal:

```bash
npx create-medusa-app@1.2.6-snapshot-20240108173057 --verbose
```
